### DISCUSSION TITLE
(0, redux.combineReducers) is not a function

### DISCUSSION THREAD

**Discussion by George-Fx** [reduxjs/react-redux#1880] [Q&A]
I ran into this problem and can't find a solution.
![Screenshot_1](https://user-images.githubusercontent.com/64652126/160423019-49e0726c-a8c9-4954-9ef5-1e215278cc78.png)

code:
`code: 
`import { combineReducers } from "redux";
import reducerOne from "./reducerOne";
import reducerTwo from "./reducerTwo";

const rootReducer = combineReducers({
    reducerOne: reducerOne,
    reducerTwo: reducerTwo,
});

export default rootReducer;
`
`

**Comment by markerikson** [comment]
This is always an indicator that there's something wrong with your build or bundling setup, or your import statements.  Please check how you're loading the Redux library and importing it.
